pkg/framework/plugins/removepodsviolatingtopologyspreadconstraint/types.go

```diff
@@ -28,7 +28,11 @@ import (
 type RemovePodsViolatingTopologySpreadConstraintArgs struct {
 	metav1.TypeMeta `json:",inline"`
 
-	Namespaces             *api.Namespaces       `json:"namespaces"`
-	LabelSelector          *metav1.LabelSelector `json:"labelSelector"`
-	IncludeSoftConstraints bool                  `json:"includeSoftConstraints"`
+	Namespaces    *api.Namespaces       `json:"namespaces"`
+	LabelSelector *metav1.LabelSelector `json:"labelSelector"`
+	Constraints   []string              `json:"constraints"`
+	NodeFit       *bool                 `json:"nodeFit,omitempty"`
+
+	// Deprecated: use Constraints list instead
+	IncludeSoftConstraints bool `json:"includeSoftConstraints"`
```

Comment:
v1alpha2 is not out yet officially so it is ok to drop the arg completely an implement the conversion from v1alpha1 -> v1alpha2